types/ui_schema/example/field_layout.ts

```diff
@@ -0,0 +1,47 @@
+import type { UISchema } from '../../mod.ts'
+
+const LoginFormUISchema: UISchema<{
+  name: string
+  password: string
+}> = {
+  'ui:submitButtonOptions': {
+    submitText: '登录',
+  },
+  name: {
+    'ui:widget': 'SingleLineWidget',
+    'ui:x-field-layout': [
```

Comment:
这里还是叫 x-layout 有没问题呀？

```suggestion
    'ui:x-layout': [
```

Reply:
这个会有冲突吧，如果是 Object 类型的字段。两者可以同时存在的

Reply:
例子里面加一下这种情况看看

Reply:
我觉得 filed 的 layout 没必要自定义呢，我看了下那里还有很多 template，我们不会都实现吧，我觉得我们实现两个固定的 template 就好了，没必要做成 layout 这种

我们为什么要做成 layout 这种方式？

Reply:
固定的两种（水平/垂直）不够表达 label 的对齐，label 的宽度这些，需要额外的参数来表达。这里想要比较好的扩展灵活性，才选择这样设计。我们不会去实现所有的 template，只需要调整 FieldTemplate 即可（所有的字段类型都会使用这个模版）